tools/generate-classes/generate.js

```diff
@@ -315,35 +315,47 @@ function generate(options, schema, writers) {
   );
 
   const jsonHandlerHeadersImpl = lodash.uniq([
+    NameFormatters.getJsonHandlerIncludeFromName(base, readerNamespace),
     ...lodash.flatten(properties.map((property) => property.readerHeadersImpl)),
   ]);
+
+  if (thisConfig.extensionName) {
+    jsonHandlerHeadersImpl.push("<CesiumUtility/ExtensibleObject.h>");
+    jsonHandlerHeadersImpl.push("<any>");
+  }
+
   jsonHandlerHeadersImpl.sort();
 
   function generateJsonHandlerOptionsInitializerList(properties, varName) {
     const initializerList = properties
       .filter((p) => p.readerType.toLowerCase().indexOf("jsonhandler") != -1)
-      .map(
-        (p) =>
-          `_${p.cppSafeName}(${p.schemas && p.schemas.length > 0 ? varName : ""
-          })`
-      )
+      .filter((p) => p.schemas && p.schemas.length > 0)
+      .map((p) => `_${p.cppSafeName}(${varName})`)
```

Comment:
What's the reason for this change? It appears to result in not all of the fields being initialized in this constructor.

Reply:
I believe it was for this check: https://clang.llvm.org/extra/clang-tidy/checks/readability/redundant-member-init.html

Reply:
Oh. Huh. That's a strange rule. I was under the impression the C++ init rules are too complicated for mere mortals to understand and therefore you should always init all fields if you init any of them, and in the order they're declared. Is there new wisdom here that I'm unaware of?

Reply:
We can always just disable the check.

Reply:
Based on a quick search, I think I got that from Effective C++. It could easily be outdated practice at this point, but it's not obvious to me why it would be.

Reply:
Ok sounds good @azrogers! I'm still interested in hearing why clang-tidy even has such a rule, if anyone knows. Presumably there's a reason for it.